src/mixins/filing-template-mixin.ts

```diff
@@ -252,7 +255,7 @@ export default class FilingTemplateMixin extends DateMixin {
     })
 
     // restore Persons and Organizations
-    this.setOrgPersonList(draftFiling.incorporationApplication.parties)
+    this.setOrgPersonList(draftFiling.incorporationApplication.parties || [])
```

Comment:
Safe fallback in case `parties` was saved as null (or not saved at all) in the draft.